webpack.config.js

```diff
@@ -1,43 +1,77 @@
+const webpack = require('webpack');
+const path = require('path');
+const HtmlWebpackPlugin = require('html-webpack-plugin');
+const CleanWebpackPlugin = require('clean-webpack-plugin');
 
-const path = require("path");
-const HtmlWebpackPlugin = require("html-webpack-plugin");
+// Our directory path
```

Comment:
@JohnMadakin can these path definitions be abstracted?

Reply:
Naming this file `webpack.config` kinda means that this is the main config. Could you rename this to reflect that it is a base config. Also, to make the codebase cleaner you can have the configs in a folder